dist/aframe-sprite-component.js

```diff
@@ -52,7 +52,10 @@
 	        },
 	        resize:{
 	            default: '1 1 1'
-	        }
+	        },
+			alpha : {
+				default : 0.4
```

Comment:
thanks! just out of curiosity why default to 0.4?

Reply:
my project works on that value lol :D oh yea i haven't minified the js yet tho